docs/rules/generator-star-spacing.md

```diff
@@ -71,7 +71,11 @@ And the option has shorthand as a string keyword:
 "generator-star-spacing": ["error", "after"]
 ```
 
-When using `{"before": true, "after": false}` this placement will be enforced:
+## Examples
```

Comment:
I've got `generator-star-spacing` flagged in my notes as needing a lot of attention - the opening sections in the file need quite a bit of clean-up which was outside the scope of this current sweep. Should I create a ticket to track?

In the interim, I've added an h2 Options to the file in PR batch 4 - but it will still need further clean-up in a later sweep.